localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -190,12 +160,18 @@ void EKFLocalizer::timerCallback()
     stop_watch_.tic();
 
     // save the initial size because the queue size can change in the loop
-    const size_t n = pose_queue_.size();
-    for (size_t i = 0; i < n; ++i) {
+    const auto t_curr = this->now();
+    for (size_t i = 0; i < pose_queue_.size(); ++i) {
```

Comment:
`pose_queue_.size()` might decrease within the loop, so it should store the size as `n` beforehand and then loop `n` times.

Reply:
Tashikani!!!!
Thanks :bow: Fixed here: https://github.com/autowarefoundation/autoware.universe/pull/5424/commits/048faf9e4767469a200a71d37957dcd39469191b